ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -81,7 +81,7 @@ RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y ccache
 ))@
 
 @[if run_abichecker]@
-RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y python3 abi-compliance-checker python3-catkin-pkg-modules
+RUN python3 -u /tmp/wrapper_scripts/apt.py update-install-clean -q -y python3-pip abi-compliance-checker python3-catkin-pkg-modules
```

Comment:
oh, the pip addition was needed when I was testing the failure on ROS1 packages

Reply:
`python3` is still needed, no? So this should just add `python3-pip` to the list and not rely on it pulling in `python3` implicitly.

Reply:
agree. Changed in b181de8